Userland/Libraries/LibWeb/CSS/Parser/Parser.h

```diff
@@ -222,6 +222,10 @@ class Parser {
 
     CSSRule* convert_to_rule(NonnullRefPtr<Rule>);
     CSSMediaRule* convert_to_media_rule(NonnullRefPtr<Rule>);
+    JS::GCPtr<CSSKeyframesRule> convert_to_keyframes_rule(Rule&);
+    JS::GCPtr<CSSImportRule> convert_to_import_rule(Rule&);
+    JS::GCPtr<CSSNamespaceRule> convert_to_namespace_rule(Rule&);
+    JS::GCPtr<CSSSupportsRule> convert_to_supports_rule(Rule&);
```

Comment:
Not sure about the return types here. The existing function return raw pointers, but the `CSSFooRule::create()` functions all return a `NonnullGCPtr`, so it seemed weird to not just return that? This code is a couple of years old though so we might not have had `GCPtr` and friends around back then.

Reply:
Yeah `GCPtr` or `NonnullGCPtr` is nicer IMO, communicates better what's being returned.

Reply:
OK, I'll update the existing ones too then. :+1: